randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "3507547160": [
+        # Lava Lake PB Sandstone + No-Respawn Tip
+        # Replaces Lava Lake sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;.\n\n"
+            "Additionally, in most game versions, these rocks will not reappear after being destroyed."
+        ),
+    ],
+    "3269646811": [
+        # Reflecting Pool PB Talloric Alloy Tip
+        # Replaces Reflecting Pool lid scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Talloric Alloy&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "1876916293": [
+        # Fungal Hall B PB Tip
+        # Replaces MQB lower right computer screen scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;The item within Fungal Hall B's mushroom ring can be uncovered with "
+            "&main-color=#FF3333;Morph Ball Bombs&main-color=#89D6FF; or "
+            "&main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "4186358462": [
+        # Life Grove Tunnel PB Tip
+        # Replaces Life Grove & LGT Bendezium rock scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;This &main-color=#FF3333;Bendezium&main-color=#89D6FF; surface has a note:\n\n\n "
+            "Life Grove and Life Grove Tunnel each contain a &main-color=#FF3333;Sandstone&main-color=#89D6FF; "
+            "structure.\n\nLike most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "they can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "323204925": [
+        # Watery Hall Access Wall Tip
+        # Replaces Watery Hall Access holobanner scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;In addition to missiles, the weak wall section in this room is able to be destoyed "
+            "by the &main-color=#FF3333;Charge Beam&main-color=#89D6FF;."
+        ),
+    ],
+    "56434148": [
+        # Central Dynamo Unlocked Door Tip
+        # Replaces Central Dynamo lower right computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        "&main-color=#89D6FF;Fighting the Cloaked Drone locks all doors except the door to Quarantine Access A.",
```

Comment:
There are different triggers that lock the doors. This can be triggered without triggering the drone fight. I would reword to indicate that the doors lock when approached after entering the room and will only unlock after defeating the drone and doing the maze.

Reply:
Just so I understand, the doors lock when you hit invisible triggers separate from the drone fight?